tests/install_test.sh

```
@@ -0,0 +1,48 @@
+    trochilidae_set_hostname("install_test");
+    trochilidae_timer_start("smoke");
+    trochilidae_timer_stop("smoke");
+    $info = trochilidae_timer_get_info("smoke");
+    assert($info["start_count"] === 1);
+    assert($info["stop_count"] === 1);
+    echo "  set_tag       OK\n";
+    echo "  set_hostname  OK\n";
+    echo "  timer         OK\n";
+    echo "  flush         OK\n";
+    echo "  All functions passed\n";
+'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)